app/frontend/src/routes.ts

```diff
@@ -130,3 +123,4 @@ export const routeToCommunity = (
 export const editCommunityPageRoute = `${communityBaseRoute}/:communityId/:communitySlug/info/edit`;
 export const routeToEditCommunityPage = (id: number, slug: string) =>
   `${routeToCommunity(id, slug, "info")}/edit`;
+export const composingDiscussionHash = "new";
```

Comment:
Do we need to update the URL document on this? Creating a new guide is currently under `guide/new`, so I wonder if those needs updating 🤔 
Or maybe not because they will actually be on a new page (although still to be decided as they haven't been implemented properly yet) unlike discussions?

Reply:
Yes my logic was it's not an actual page, so a hash is fine.